diff --git a/src/api-client/setup.cpp b/src/api-client/setup.cpp
index 33a4ac8..8b6d19b 100644
--- a/src/api-client/setup.cpp
+++ b/src/api-client/setup.cpp
@@ -22,7 +22,7 @@ void addSetupHeaders(HTTPClient &https, ApiSetupInputs &inputs)
 ApiSetupResult fetchApiSetup(ApiSetupInputs &apiSetupInputs)
 {
   return withHttp(
-      apiSetupInputs.baseUrl + "/api/setup/",
+      apiSetupInputs.baseUrl + "/api/setup",
       [&apiSetupInputs](HTTPClient *https, HttpError error) -> ApiSetupResult
       {
         if (error == HttpError::HTTPCLIENT_WIFICLIENT_ERROR)
